### ISSUE TITLE
Setting `lsp-dart-flutter-fringe` to `nil` does not disable fringe colors

### ISSUE THREAD

**Issue by zoechi** [emacs-lsp/lsp-dart#10]
**Describe the bug**

I tried to disable fringe colors, but it didn't work.

I tried

`M-x lsp-dart-flutter-fringe-colors-mode`, `(setq lsp-dart-flutter-fringe-colors nil)`, and
```lisp
(use-package lsp-mode
  :ensure t
  :hook ((dart-mode . lsp))
  :config (setq lsp-dart-flutter-fringe-colors nil))
```
 
but the fringe colors show the same.

**Note**
Before logging the bug please make sure that the issue is reproducible with latest melpa packages and it is reproducible with the clean config(the config from the getting started page.

**To Reproduce**
{populate}

**Expected behavior**
{populate}

**Screenshots**

**Logs**
Please include the debug stack trace (if there is an error) and the content of Messages buffer with `lsp-print-io` set to t in case the bug is related to client->server communication.

```
GNU Emacs 28.0.50 (build 4, x86_64-pc-linux-gnu, GTK+ Version 3.24.5, cairo version 1.16.0) of 2020-04-03
```
```
lsp-dart           20200422.36
```

**Comment by zoechi** [comment]
Seems to at least require a restart of Emacs to take effect. 
Setting the var and close/reopen the buffer doesn't have any effect.

**Comment by ericdallo** [comment]
Thank you for the report @zoechi, it was a bug when disabling the `lsp-dart-flutter-fringe-colors-mode` that doesn't clean the colors in the current buffer. 

I released a new [tag](https://github.com/emacs-lsp/lsp-dart/releases/tag/1.6.6) fixing it today.

Feel free to reopen if the problem persist.

**Comment by ericdallo** [comment]
the variable `lsp-dart-flutter-fringe-colors` is checked only when initializing the LSP workspace for a project, so changing it in an already open buffer will not work, that's why restarting Emacs solved. For a realtime disable/enable the `lsp-dart-flutter-fringe-colors-mode` will do the trick.

**Comment by zoechi** [comment]
`lsp-dart-flutter-fringe-colors-mode` didn't work either, not sure if you meant this was supposed to be fixed by the new tag.
I also tried restarting lsp and closing/reopening the file.

**Comment by ericdallo** [comment]
Yeah, this was supposed to be fixed in the `1.6.6` tag, the bug keeps happening after the `1.6.6` tag?

**Comment by zoechi** [comment]
Thanks, now `M-x lsp-dart-flutter-fring-colors-mode` removes the colors from the fringe.
